File: .vscode/IFAL/Python/[#27].py
"""[#2] Apresente o valor de uma temperatura em graus Celsius e Fahrenheit. A função deve
ler a temperatura em graus Kelvin;"""

def graus_C (temp):
	print (temp - 273.15)
def graus_F (temp):
	print ((temp - 273.15) * 1.8 + 32)
    
temp = float(input("Digite a temperatura (K): "))

print("\nA temperatura em °C é: "), graus_C(temp)
print("A temperatura em F é: "), graus_F(temp)

File: programas_C/[#27].c
/*[#2] Apresente o valor de uma temperatura em graus Celsius e Fahrenheit. A função deve
ler a temperatura em graus Kelvin;*/
#include <stdio.h>

void graus_C(float temp){
    printf ("A temperatura em graus °C é: %f", temp - 273.15);
}
void graus_F(float temp){
    printf ("A temperatura em graus °F é: %f", (temp - 273.15) * 1.8 + 32);
}

int main(){
    float temp;

    printf("Digite a temperatura (K): ");
    scanf("%f", &temp);

    printf("\n\n");
    graus_C(temp);
    printf("\n");
    graus_F(temp);

    return 0;
}